src/GitHub/Endpoints/PullRequests/Comments.hs

```diff
@@ -8,6 +8,8 @@
 module GitHub.Endpoints.PullRequests.Comments (
     pullRequestCommentsIO,
     pullRequestCommentsR,
+    pullRequestCommentsWithReactionsIO,
+    pullRequestCommentsWithReactionsR,
```

Comment:
export from `GitHub` module as well.

Reply:
And exported `pullRequestCommentsIO` to boot.